java/fury-core/src/main/java/io/fury/serializer/Serializers.java

```diff
@@ -420,12 +420,21 @@ public StringBuilderSerializer(Fury fury) {
 
     @Override
     public void write(MemoryBuffer buffer, StringBuilder value) {
-      stringSerializer.writeJavaString(buffer, value.toString());
+      int length = value.length();
+      buffer.writeInt(length);
+      for (int i = 0; i < length; i++) {
+        buffer.writeChar(value.charAt(i));
```

Comment:
`charAt` is expensive in JDK9+ since internal `java.lang.AbstractStringBuilder` use `byte[]` instead for jdk9+.

Can we extract internal `char[]` for jdk8 and `byte[]` for jdk9+ to serialize instead?

Reply:
We did same thing in `StringSerializer`

Reply:
yup, i try to refer